.travis.yml

```diff
@@ -18,9 +18,23 @@ addons:
     packages:
     - python3-pip
 
+before_install:
+  - if [ ! -d $HOME/gcloud/google-cloud-sdk ]; then
+        mkdir -p $HOME/gcloud &&
+        wget https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz --directory-prefix=$HOME/gcloud &&
+        cd $HOME/gcloud &&
+        tar xzf google-cloud-sdk.tar.gz &&
+        printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh &&
```

Comment:
you might be able to use the `--quiet` flag here rather than piping canned answers in - that's a little brittle.

Reply:
Also I'm not 100% certain about the interaction between the compound `&& ... && ... &&` and a pipe with a compound command on the other side...

Reply:
Added `-q` and removed the `&&`'s